taipy/gui/gui.py

```diff
@@ -2346,7 +2346,7 @@ def _bind_custom_page_variables(self, page: CustomPage, client_id: t.Optional[st
                     if (
                         (not page._binding_variables or k in page._binding_variables)
                         and not k.startswith("_")
-                        and not isinstance(v, (ModuleType, FunctionType))
+                        and not isinstance(v, ModuleType)
```

Comment:
How do you deal with functions that are in the scope ?
I saw json messages ...

Reply:
There is a filter implemented in taipy-designer for the json adpater. The message wont be shown there.